common/txmgr/types/config.go

```diff
@@ -48,6 +44,10 @@ type ConfirmerConfig interface {
 	FeeBumpPercent() uint16
 }
 
+type ConfirmerEvmConfig interface {
```

Comment:
This package, or anything under common/... is supposed to be generic for any chain.
The types here should be bae types which can be extended by chain-families in their own package.

Thus, could you rename fields under this path, to generic names, and not EVM?

ConfirmerConfig, or ConfirmerRPCConfig, or similar.
Don't want to have EVM name here